indico/modules/events/editing/models/editable.py

```diff
@@ -91,3 +92,11 @@ def event(self):
     def can_comment(self, user):
         return (self.event.can_manage(user, permission='paper_editing')
                 or self.contribution.is_user_associated(user, check_abstract=True))
+
+    def review_conditions_ok(self):
+        review_conditions = editing_settings.get(self.event, 'review_conditions').values()
+        file_types = [file.file_type_id for file in self.revisions[-1].files]
+        for review_condition in review_conditions:
+            if all(file_type in file_types for file_type in review_condition):
+                return True
+        return False
```

Comment:
`return all(...)` with two loops inside the `all()`. This also ensures that not having any conditions will be considered OK, since right now you;d return False in this case.